### DISCUSSION TITLE
Are you still helping with the Galaxy Brain achievement?

### DISCUSSION THREAD

**Discussion by Johnmiicheal** [drknzz/GitHub-Achievements#56] [Q&A] [answered]
Hello, please I'd like to get the Galaxy brain achievement. I don't know if anyone's still helping with that

**Comment by S1i** [comment, marked as answer]
I don't even know if you can anymore 😭, I've been trying for a little bit and have answered  6+ discussions.

**Reply by drknzz** [reply]
Seems a bit weird that it does not show up. Have you tried answering your own discussion previously? There were some cases where that broke the achievement.

**Reply by S1i** [reply]
I have answered discussions on my own repos & other repos but have deleted my own repos where I answered discussions due to it not working
